packages/cli/src/__tests__/commands/platform/PlatformCommand.test.ts

```diff
@@ -1,17 +1,17 @@
 import { isError } from '@prisma/internals'
 
-import { PlatformCommand } from '../../../platform/PlatformCommand'
+import { $ } from '../../../platform/$'
 
 describe('--early-access-feature flag', () => {
   it('should not work without it', () => {
     // eslint-disable-next-line
```

Comment:
Why is `// eslint-disable-next-line` needed?

Reply:
![CleanShot 2023-11-21 at 13 41 58@2x](https://github.com/prisma/prisma/assets/284476/c04f0dab-8842-4bb3-b848-dfb2cd91ce51)